americangut/per_category.py

```diff
@@ -26,11 +28,83 @@ def cat_taxa_summaries():
         for col in table.ids():
             if col == 'Unknown':
                 continue
-            cleaned_col = col.split('(')[0].strip().replace(' ', '_')
+            # Clean the string to just the base information of the response
+            # and url compliant names.
+            cleaned_col = col.split('(')[0].strip().replace(
+                ' ', '_').rstrip('+')
             filename = '-'.join([name, cleaned_col]) + '.txt'
 
             with open(join(out_dir, filename), 'w') as f:
                 for otu, val in zip(ids, table.data(col)):
                     if val == 0.0:
                         continue
                     f.write('%s\t%s\n' % (otu, str(val)))
+
+
+def cat_alpha_plots():
```

Comment:
Hesitant to do MD5 because it could be sliiiightly rendered differently depending on the system tests are run on. Checking file is output would work though, and also makes sure the proper naming for files is there so the website can use them. Adding that.